domains/Discussions/Controllers/QuestionsUpdateController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class QuestionsUpdateController extends Controller
+{
+    private Question $questions;
+
+    public function __construct(Question $questions)
+    {
+        $this->questions = $questions;
+    }
+
+    public function __invoke(int $questionId, Request $request): Response
+    {
+        $question = $this->questions->newModelQuery()->findOrFail($questionId);
```

Comment:
We don't need to call the `newModelQuery()`, here.

Reply:
we don't :)